index.bs

```diff
@@ -12680,6 +12735,11 @@ class Vec3 {
 
 <h2 id="changes">
 Change Log</h2>
+<h3 id="candidate-changes-1">
+  Candidate Changes since Recommendation of 17 Jun 2021
+</h3>
+  * <a href="https://github.com/WebAudio/web-audio-api/pull/2361">PR 2361</a>: Use new Web IDL buffer primitives
```

Comment:
This correction used 7 correction boxes to make it easy to see that something was changed.  Perhaps it would make sense to link to all 7 boxes here in the change log so there's an easy way to find all 7?